Makefile

```
@@ -22,13 +22,15 @@ default: all
 
 .PHONY: all external-table fdw cli server install install-server stage tar rpm rpm-tar deb deb-tar clean test it help
 
-all: external-table cli server
+all: external-table cli server fdw
```

```suggestion
all: fdw external-table cli server
```
let's have C-side projects first, then the rest